spec/helpers/ems_cloud_helper/textual_summary_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "spec_helper"
+
+describe EmsCloudHelper do
+  before do
+    controller.send(:extend, EmsCloudHelper)
+    self.class.send(:include, EmsCloudHelper)
```

Comment:
Leave this; I'll change it in a follow-up. We currently don't have many (any?) good examples of The Right Way to spec helpers; I think most of our existing specs do something like this.